modules/assim.sequential/R/Analysis_sda.R

```diff
@@ -235,7 +237,7 @@ GEF<-function(settings, Forecast, Observed, H, extraArg, nitr=50000, nburnin=100
       Rmcmc_tobit2space <- buildMCMC(conf_tobit2space)
       
       #restarting at good initial conditions is somewhat important here
-      Cmodel_tobit2space <- compileNimble(tobit2space_pred)
+      Cmodel_tobit2space <- compileNimble(tobit2space_pred, showCompilerOutput = TRUE)
```

Comment:
Is `showCompilterOutput = TRUE` a good idea here? IME, Nimble output is pretty verbose. If it's useful, we can keep it -- just checking that this isn't an artifact of interactive debugging.

Reply:
It was definitely an artifact of debugging! I removed it- thanks for catching that!